kolibri/core/auth/management/commands/sync.py

```diff
@@ -475,11 +479,14 @@ def handler(transfer_session):
             """
             :type transfer_session: morango.models.core.TransferSession
             """
-            progress = (
-                100
-                * transfer_session.records_transferred
-                / float(transfer_session.records_total)
-            )
+            try:
+                progress = (
+                    100
+                    * transfer_session.records_transferred
+                    / float(transfer_session.records_total)
+                )
+            except ZeroDivisionError:
+                progress = 100
```

Comment:
Not sure whether this was fixed elsewhere, but I was encountering divzero errors in my tests, when no data was being synced.